File: aoj/volume100/csharp/A10012.cs
using System;

class AOJ
{
	static void Main()
	{
		while (true)
		{
			var input = Console.ReadLine().Split();
			var h = int.Parse(input[0]);
			var w = int.Parse(input[1]);
			if (h == 0 && w == 0)
				break;

			for (int hi = 0; hi < h; hi++)
			{
				for (int wi = 0; wi < w; wi++)
				{
					Console.Write("#");
				}
				Console.WriteLine();
			}
			Console.WriteLine();
		}
	}
}

File: aoj/volume100/scala/A10012.scala
object Main {
  def main(args: Array[String]) {
    loop
  }

  def loop {
    val input = scala.io.StdIn.readLine split " "
    val h = input(0).toInt
    val w = input(1).toInt
    if (h == 0 && w == 0) {
      return
    }

    val rect = ("#"*w+"\n")*h
    println(rect)
    loop
  }
}
